main/models.py

```diff
@@ -17,6 +17,15 @@ class Token(models.Model):
     confirmation_limit = models.IntegerField(default=0)
     decimals = models.IntegerField(default=0)
 
+    token_ticker = models.CharField(max_length=200)
+    token_type = models.IntegerField(default=1)
+    nft_token_group = models.ForeignKey(
+        "main.Token",
+        on_delete=models.CASCADE,
+        related_name='transactions',
```

Comment:
Please change the related_name to `children`.

Reply:
Done

Reply:
Looks like we have to generate a migration file for that!. I did one in here 7b004970d0dd2ce6b09a2cbea041a9a7e5fbe754  since this was already closed.